database/src/main/postgres/runs/V1.5.11__get_partitioning_measures.sql

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+ CREATE OR REPLACE FUNCTION runs.get_partitioning_measures(
+     IN i_partitioning          JSONB,
+     OUT measure_name           TEXT,
+     OUT measured_columns       TEXT[],
+     OUT status                 INTEGER,
+     OUT status_text            TEXT
+ ) RETURNS SETOF record AS
+$$
+-------------------------------------------------------------------------------
+--
+-- Function: runs.get_partitioning_measures(2)
+--      Iterates over a JSONB object and returns each key-value pair as a record
+--
+-- Parameters:
+--      i_partitioning      - JSONB object where each key is a measure name and its corresponding value is an array of measured columns
+--
+-- Returns:
+--      measure_name        - Name of the measure
+--      measured_columns    - Array of columns associated with the measure
+--
+-------------------------------------------------------------------------------
+
+DECLARE
+    _fk_partitioning                    BIGINT;
+BEGIN
+    _fk_partitioning = runs._get_id_partitioning(i_partitioning);
+
+    IF _fk_partitioning IS NULL THEN
+        measure_name := 'Partitioning not found';
+        measured_columns := '{}';
+        status := 41;
+        status_text := 'The partitioning does not exist.';
+        RETURN;
+    END IF;
+
+    status = 11;
```

Comment:
I would consider implementing another (third) status code, for situation when there are no measures in DB yet

Reply:
Based on this it will be something in range 40-49.....perhaps 42: https://github.com/AbsaOSS/fa-db/blob/master/core/src/main/scala/za/co/absa/fadb/status/README.md

What do you think @benedeki ?

Reply:
The third status code might be useful for handling in fa-db Or actually might be complicating it. I am not sure in the current state of fa-db lib 😊 
But if a third status code is introduced, it should definitely be of the OK status range 10-19 ( I would suggest 16.). Because it's totally OK not to have any measures defined.

Reply:
Yah, it makes sense that we can have a partitioning without measures associated with it.